src/test/java/com/epam/tc/hw7/JdiSiteTest.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.tc.hw7;
+
+import static com.epam.tc.hw7.pages.JdiHomePage.ROMAN_IOVLEV;
+
+import org.testng.annotations.Test;
+
+public class JdiSiteTest extends AbstractTest {
+
+    @Test(dataProvider = "DataProviderJson", dataProviderClass = DataProviderMetalsAndColors.class)
+    public void jdiHomepageTest(jdi.entities.MetalsAndColors metalsAndColors) {
```

Comment:
Not the best test name. Try to come up with something more explicitly describing the essence.

Reply:
Added description and renamed test for HM7